tests/testthat/test-plotly.R

```diff
@@ -317,3 +317,38 @@ test_that("toWebGL() shouldn't complain if it's already webgl", {
     toWebGL()
   expect_silent(plotly_build(p))
 })
+
+test_that("Line breaks are properly translated (R -> HTML)", {
+  skip_if_not_installed(pkg = "forcats")
+  
+  # create target labels
+  suffix <- "\n\n(third line)\n(fourth line)"
+  
+  target_labels <- iris$Species %>%
+    unique() %>%
+    as.character() %>%
```

Comment:
I think you can get rid of the `as.character()` here as `paste()` and `gsub()` will coerce to character

Reply:
Ok, done.